src/tests/nativeaot/SmokeTests/ControlFlowGuard/ControlFlowGuard.csproj

```diff
@@ -6,7 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ControlFlowGuard>Guard</ControlFlowGuard>
     <Optimize>true</Optimize>
-    <CLRTestTargetUnsupported Condition="'$(TargetsWindows)' != 'true'">true</CLRTestTargetUnsupported>
+    <CLRTestTargetUnsupported Condition="'$(TargetsWindows)' != 'true' or '$(TargetArchitecture)' != 'x64'">true</CLRTestTargetUnsupported>
```

Comment:
Could you file an issue on this and link it here? I pulled down the crash dump and this is failing on invalid instruction when running code generated here:

https://github.com/dotnet/runtime/blob/ce719aa21f73ff387956b2f2ab148d32a9adaed7/src/tests/nativeaot/SmokeTests/ControlFlowGuard/ControlFlowGuard.cs#L167-L200

I don't immediately see what's wrong with it. The instruction disassembles into `ret` fine in the crash dump. The test is crashing before we enable CFG enforcement. Maybe I'm missing some additional Windows ARM64 knowledge. I don't quickly know a way to fix this test (maybe @jkotas would know what's wrong with it), so let's track it with an issue. CFG should work.

Reply:
One obvious problem is missing `FlushInstructionCache` call.

Reply:
> One obvious problem is missing `FlushInstructionCache` call.

Thanks! Likely another couple of hours of troubleshooting saved! :)